src/OrchardCore/OrchardCore.Abstractions/Shell/Models/FeatureProfile.cs

```diff
@@ -9,7 +9,7 @@ public class FeatureProfile
 
         public string Name { get; set; }
 
-        public List<FeatureRule> FeatureRules = [];
+        public List<FeatureRule> FeatureRules { get; set; } = [];
```

Comment:
```suggestion
        public List<FeatureRule> FeatureRules { get; init; } = [];
```

Reply:
@MikeAlhayek , This can cause some code to compile with errors, such as
```c#
profile.FeatureRules = JsonConvert.DeserializeObject<List<FeatureRule>>(model.FeatureRules);
```

Reply:
Why do you need to serialize it publicly?

Reply:
It's because that's what the existing code does
https://github.com/OrchardCMS/OrchardCore/blob/f53d16c5854817b6c540f84c90c590a9eef85783/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/FeatureProfilesController.cs#L160-L167

Reply:
Okay.